python/build.xml

```diff
@@ -51,19 +51,25 @@
 	</target>
 	
 	
+	
 	<target name="init-env" unless="python.nobuild">
 		<dirname property="python.basedir" file="${ant.file.python}"/>
 		
 		<property name="python.out.path" value="${python.basedir}/bin/ant" />
 		<property name="python.out.classes" value="${python.out.path}/classes"/>
 		<property name="python.sources" value="${python.basedir}/source"/>
 		
+		
 		<path id="python.dependencies.path">
 			<pathelement location="${core.out.classes}"/>
 			
 			<fileset dir="${eclipse.home}/plugins/">
 				<include name="org.eclipse*.jar"/>
 			</fileset>
+			
+			<fileset dir="${eclipse.home}/plugins/">
+				<include name="com.python.pydev.debug_*/debug.jar"/>
+			</fileset>
```

Comment:
When the build.xml of build module is run to compile, it searches for the jar file which contais the 'com.python.pydev.debug'. The fileset tag which has been added gives the location to this package, without this extra fileset tag, the build will fail.